include/deal.II/non_matching/fe_values.h

```diff
@@ -689,7 +693,7 @@ namespace NonMatching
      * This container is a std::deque, which is compatible with the
      * `FEInterfaceValues` class that does not have a copy-constructor.
      */
-    std::deque<std::optional<dealii::FEInterfaceValues<dim>>>
+    std::optional<dealii::FEInterfaceValues<dim>>
       fe_values_inside_full_quadrature;
```

Comment:
The comment now no longer matches the data type. In fact, the whole comment talks about "For each element", but you now only store a single object.